src/components/CreateAccountPage.js

```diff
@@ -21,31 +74,31 @@ function CreateAccountPage () {
         <Grid item xs={12}>
           <Grid container spacing={2}>
             <Grid item>
-              <TextField id="inpFirstName" label="First Name" variant="filled" required value={firstName} onChange={(event) => {
+              <TextField id="inpFirstName" label="First Name" variant="filled" error={firstNameError} helperText={firstNameError} required value={firstName} onChange={(event) => {
                 setFirstName(event.target.value);
               }}/>
             </Grid>
             <Grid item>
-              <TextField id="inpLastName" label="Last Name" variant="filled" required value={lastName} onChange={(event) => {
+              <TextField id="inpLastName" label="Last Name" variant="filled" error={lastNameError} helperText={lastNameError} required value={lastName} onChange={(event) => {
                 setLastName(event.target.value);
               }}/>
             </Grid>
           </Grid>
         </Grid>
         <Grid item xs={12}>
-          <TextField id="inpEmail" label="Email" variant="filled" required value={email} onChange={(event) => {
+          <TextField id="inpEmail" label="Email" variant="filled" error={emailError} helperText={emailError} required value={email} onChange={(event) => {
```

Comment:
Looks and works great! Just one thing, can you make the email text field as wide as both first name and last name fields. You can do this by adding sx property to text field 
sx={{ width: '500px' }}

Reply:
Feel free to merge after adding this